### ISSUE TITLE
Increased plasma farnesol

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5847]
**Preferred term label:**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6037199/

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
